Libraries/DirectMLX.h

```diff
@@ -722,10 +745,14 @@ namespace dml
             // number of input nodes on the graph (e.g. in the case of unused empty inputs), but never smaller.
             assert(inputCount == 0 || inputCount >= graph.inputCount);
 
-            std::vector<DML_GRAPH_NODE_DESC> graphNodes(graph.nodes.size());
-            for (size_t i = 0; i < graphNodes.size(); ++i)
+            std::vector<DML_GRAPH_NODE_DESC> graphNodes(graph.operatorNodes.size() + graph.constantNodes.size());
+            for (size_t i = 0; i < graph.operatorNodes.size(); ++i)
+            {
+                graphNodes[i] = { DML_GRAPH_NODE_TYPE_OPERATOR, &graph.operatorNodes[i] };
+            }
+            for (size_t i = 0; i < graph.constantNodes.size(); ++i)
             {
-                graphNodes[i] = { DML_GRAPH_NODE_TYPE_OPERATOR, &graph.nodes[i] };
+                graphNodes[i + graph.operatorNodes.size()] = {DML_GRAPH_NODE_TYPE_CONSTANT, &graph.constantNodes[i]};
```

Comment:
```suggestion
                graphNodes[i + graph.operatorNodes.size()] = { DML_GRAPH_NODE_TYPE_CONSTANT, &graph.constantNodes[i] };
```
(consistent with ` { DML_GRAPH_NODE_TYPE_OPERATOR, &graph.operatorNodes[i] };`, or make line 751 the same)